### PR TITLE
Cherry-pick from 2.2 GDB-8232: Removes the url parameters when the sparql…

### PR DESCRIPTION
**Pull request by boyan-tonchev**
… view is loaded

## What
When load "sparql" view by "Welcome page saved queries", "Saved queries" or "Get URL to current query" functionality a new tab is opened with filled query. When the tab is closed and page is reloaded, the tab appears again.

## Why
The url of sparql view contains parameters that activates functionality that opens a new tab, loads a query and fill it into the opened tab.

## How
Removes all parameters when page is loaded

### CODE REVIEW

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

File: src/js/angular/core/directives/queryeditor/query-editor.directive.js [hunk 1]
```diff
@@ -875,6 +866,7 @@ function queryEditorDirective($timeout, $location, toastr, $repositories, Sparql
                     scope.currentQuery.sameAs = toBoolean(sameAs);
                 }
                 autoexecuteQueryIfRequested();
+                $location.search({});
```
Comment (svilenvelikov) [inline src/js/angular/core/directives/queryeditor/query-editor.directive.js:869]: If you do this, then the previous url location would remain in the browser history which will be bad for the user experience.

Comment (boyan-tonchev) [reply src/js/angular/core/directives/queryeditor/query-editor.directive.js:869]: fixed

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**